i18n(ja) Add JA texts to 6 files in Plugins section (part 2)

J1-takai opened the pull request:
<!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->

#### Description
i18n(ja): Add Japanese Texts to Plugins section (part 2)
- 6 files from **Deep-Linking** to **LocalHost** were translated into Japanese 
- 2nd 6 documents from the Plugins, following the PR #3500.
- Closes #

J1-takai committed:
i18n(ja) Add JA texts to 6 files in Plugins section (part 2)

J1-takai committed:
Fix Japanese translation for plugin description

Rephrased the line 20.

J1-takai committed:
Fix translation note for plugin description in Japanese

Rephrased the line 18.

J1-takai committed:
Refine Japanese translation note for plugin section

Rephrased the line 18.

J1-takai committed:
Fix translation note for plugin description in Japanese

Rephrased the line 18.

J1-takai committed:
Fix translation note wording in Japanese documentation

Rephrased the line 18.

J1-takai committed:
Update Japanese translation for plugin description

Rephrased the line 18.

vasfvitor committed:
Merge branch 'v2' into plugins-2

J1-takai commented:
@vasfvitor Thank you for your continued support and thank you again for your another review/approval on PR #3625.  By the way, I have a question on this PR (#3546) and PR #3584 : Is there any action I need to  take on them?

vasfvitor commented:
@J1-takai 

https://app.netlify.com/projects/tauri-v2/deploys/69135c69ff16110008aa1d35#L2188 (building tab, line 2188 onwards)

if you can manage to fix the links, for example in this current PR
```
12:57:58 PM: 15:57:58 [ERROR] ✗ Found 6 invalid links in 4 files.
12:57:58 PM: 15:57:58 ▶ ja/develop/resources/
12:57:58 PM: 15:57:58   └─ /ja/plugin/file-system/#scopes - invalid hash
12:57:58 PM: 15:57:58 ▶ ja/plugin/dialog/
12:57:58 PM: 15:57:58   ├─ #build-an-ask-dialog - invalid hash
12:57:58 PM: 15:57:58   ├─ #build-a-message-dialog - invalid hash
12:57:58 PM: 15:57:58   └─ #build-a-file-selector-dialog - invalid hash
12:57:58 PM: 15:57:58 ▶ ja/plugin/deep-linking/
12:57:58 PM: 15:57:58   └─ #実行時にデスクトップ-Deep-Linkを登録する - invalid hash
12:57:58 PM: 15:57:58 ▶ ja/learn/security/using-plugin-permissions/
12:57:58 PM: 15:57:58   └─ /ja/plugin/file-system/#setup - invalid hash
```

it is easier when running in dev mode (pnpm dev) and you navigate to the actual hash and copy it, that is mostly what needs to be done here and probably on the other PR

J1-takai committed:
Correction of invalid hash in 3 links in ##Usage section

Links to ### Rust  in ## Usage section are corrected - EN texts are removed and replaced with JA texts.

J1-takai committed:
Change heading from 'Scopes（有効範囲）' to 'Scopes'

Correction of the header based on the Link text already used in the resource.mdx file (Removed the JA text in brackets).

J1-takai commented:
@vasfvitor  - I tried to correct/update the link errors according to the Netlify report above:

- ja/plugin/file-system/#scopes - invalid hash: Amended.
- ja/plugin/dialog/ - invalid hash: three links are corrected.
- ja/plugin/deep-linking/ - invalid hash: This Japanese link looks working properly in my draft, I do not know where to correct ...
- ja/learn/security/using-plugin-permissions/ - the link to the file-system in this file was old (the file-system.mdx is a newer translation).  I will update the links in the "using-plugin-permissions" file and issue a PR so that the links  work properly.

vasfvitor committed:
fix anchors

vasfvitor committed:
format

vasfvitor commented:
it's ok, it seems the deep-link was the case D -> d.

J1-takai commented:
@vasfvitor Thank you for your further checks/modifications. No "Upper Case" in Link-texts! - Noted! I will check the PR #3584 files.
